src/pykeen/nn/init.py

```diff
@@ -153,17 +196,11 @@ def create_init_from_pretrained(pretrained: torch.FloatTensor) -> Initializer:
             ),
         )
     """
-
-    def init_from_pretrained(x: torch.FloatTensor) -> torch.FloatTensor:
-        """Initialize tensor with pretrained weights."""
-        if x.shape != pretrained.shape:
-            raise ValueError(f"shape of pretrained {pretrained.shape} does not match shape of tensor {x.shape}")
-        return pretrained
-
-    return init_from_pretrained
+    warnings.warn(message="Please directly use `PretrainedInitializer`.", category=DeprecationWarning)
```

Comment:
we could also directly remove this, since it was not yet included in any release (afaik)

Reply:
Just remove it

Reply:
4ee9e01